content/en/monitors/guide/division-in-monitor-evaluations.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Division in Monitor Evaluations
+kind: guide
+aliases:
+  - /monitors/faq/Division-in-Monitor-Evaluations
+---
+
+## Overview
+
+Creating an alert based on a query with division is a common practice. There are some tools and behaviors that should be considered to ensure monitor settings are appropriate for evaluating these queries as intended.
```

Comment:
To reduce confusion, I think it would be best to focus on count division queries (e.g. error rates) in this article and to move the other things to separate articles.